.cirrus.yml

```diff
@@ -0,0 +1,34 @@
+env:
+  CIRRUS_CLONE_DEPTH: 1
+
+freebsd_task:
+  matrix:
+    - name: FreeBSD 13.0 (GCC 10 from packages)
+      # RunCatchingExceptionsOtherException fails on 13.0 with system Clang
+      # 11.0 and ports Clang 10/11 as well, so GCC 10 is used instead.
+      freebsd_instance:
+        image_family: freebsd-13-0
+      preinstall_script:
+        # Stock clang11 fails some exception unit tests
+        pkg install -y gcc10
+      env:
+        CC: gcc10
+        CXX: g++10
+    - name: FreeBSD 12.2 (System Clang 10)
+      freebsd_instance:
+        image_family: freebsd-12-2
+    - name: FreeBSD 11.4 (System Clang 10)
+      freebsd_instance:
+        image_family: freebsd-11-4
+  install_script:
+    pkg install -y automake autoconf libtool
+  compiler_version_script:
+    ${CXX:-"c++"} --version
+  autoreconf_script:
+    - cd c++ && autoreconf -i
+  configure_script:
+    - cd c++ && ./configure
```

Comment:
Hey sorry, I should have looked more closely at this earlier. Can you use `super-test.sh` like all the other Unix CI targets, rather than building manually? It's specifically intended to be our CI script.

Reply:
Though if it's easier to merge this now and get CI configured before iterating on that, I'm OK with that... but we should have this use `super-test.sh` long-term.

Reply:
@kentonv I gave it a brief try and it looks like super-test.sh has some bash-only features and also fails to detect system Clang for some reason on FreeBSD. Therefore, I suggest perhaps flushing this PR since it affects other PRs in pipeline and I'll open a new one to switch Cirus to use super-test.sh once I have a chance to massage it to complete on FreeBSD. Thanks!